packages/.gitignore

```diff
@@ -0,0 +1,13 @@
+# Files and directories created by pub
+.dart_tool/
+.packages
+
+# Omit commiting pubspec.lock for library packages:
+# https://dart.dev/guides/libraries/private-files#pubspeclock
+pubspec.lock
```

Comment:
Then we're supposed to commit `pubspec.lock`, right?

Reply:
I don't know why there is this new file actually 🤨
I will remove this one